front/src/applications/stdcm/types.ts

```diff
@@ -0,0 +1,4 @@
+import { ValueOf } from 'utils/types';
+import STDCM_REQUEST_STATUS from './consts';
+
+export type StdcmRequestStatusValues = ValueOf<typeof STDCM_REQUEST_STATUS>;
```

Comment:
I think StdcmRequestStatus is enough as we know that it is a type
```suggestion
export type StdcmRequestStatus = ValueOf<typeof STDCM_REQUEST_STATUS>;
```

Reply:
done